src/main/java/fr/techad/edc/popover/internal/swing/components/Popover.java

```diff
@@ -100,6 +99,18 @@ public void windowLostFocus(WindowEvent e) {
         });
     }
 
+    public void hidePopover(){
+        this.addMouseListener(new MouseAdapter() {
+            @Override
+            public void mouseExited(MouseEvent e) {
+                Rectangle popoverBounds = e.getComponent().getBounds();
+                boolean contain = popoverBounds.contains(e.getXOnScreen(), e.getYOnScreen());
+
+                Popover.this.setVisible(contain);
```

Comment:
Hide only if this option is selected

Reply:
I have already made this check at the time of the call in the IconButtonListener file because I do not have access to the helpConfiguration in the popover file.